src/test/java/org/llorllale/cactoos/matchers/AssertionTest.java

```diff
@@ -69,19 +63,20 @@ public void affirmIfResultMatches() {
      */
     @Test
     public void refuteIfResultDoesNotMatch() throws IOException {
-        this.exception.expect(AssertionError.class);
-        this.exception.expectMessage(
+        Assertions.assertThrows(
```

Comment:
@victornoel You don't think better use `Throws`here instead of `Assertions.assertThrows()` which is as static method?

Reply:
@fabriciofx this is an eggs and chicken problem, I would need `Assertion` to test assertion and it seemed a bad idea. Do you think it's not going to create problems and have tests that passes even though they should be failing? I'm not sure

Reply:
@victornoel I agree. Let's keep it.